app/src/main/java/cm/aptoide/pt/install/InstallManager.java

```diff
@@ -240,19 +241,25 @@ private boolean isAppInstalling(List<RoomInstalled> installingAppList, String pa
   }
 
   public Completable install(RoomDownload download) {
-    return install(download, false, false, true);
+    boolean shouldForceDefaultInstall =
```

Comment:
Why not use the settingsProvider you created here? You have a method for isEnforceNativeInstaller() and you wouldn't need to extract the variable either